packages/mds-db/client.ts

```diff
@@ -84,11 +84,11 @@ export async function getWriteableClient(): Promise<MDSPostgresClient> {
 
 /* eslint-reason ambigous helper function that wraps a query as Readonly */
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-export async function makeReadOnlyQuery(sql: string, values?: (string | number)[]): Promise<any[]> {
+export async function makeReadOnlyQuery(sql: string): Promise<any[]> {
```

Comment:
Why did all the SqlVals stuff get undone?

Reply:
It wasn't working properly, we need to do some debugging to figure out why the dollar sign values weren't getting overwritten